include/pyclient.h

```diff
@@ -720,6 +720,172 @@ class PyClient
         */
         void save(std::vector<std::string> addresses);
 
+        /*!
```

Comment:
Are we missing the ``use_list_ensemble_prefix`` method in this file?

Reply:
Good catch!